ui/app/pages/create-account/connect-hardware/account-list.js

```diff
@@ -47,18 +58,24 @@ class AccountList extends Component {
   }
 
   renderHeader() {
-    const { device } = this.props;
     return (
       <div className="hw-connect">
         <h3 className="hw-connect__unlock-title">
-          {`${this.context.t('unlock')} ${this.capitalizeDevice(device)}`}
+          {this.context.t('selectAnAccount')}
         </h3>
-        {device.toLowerCase() === 'ledger' ? this.renderHdPathSelector() : null}
         <h3 className="hw-connect__hdPath__title">
           {this.context.t('selectAnAccount')}
         </h3>
         <p className="hw-connect__msg">
-          {this.context.t('selectAnAccountHelp')}
+          {this.context.t('selectAnAccountHelp', [
+            <span
```

Comment:
We should not be using `span` for clickable things if at all possible to avoid. We can use something like an `<a>` element instead, so that it has better accessibility properties built-in.

Reply:
Good point, changed to button.